src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageStrings.cs

```diff
@@ -207,6 +207,9 @@ public static string Option_Collapse_usings_on_file_open
         public static string Option_Collapse_metadata_on_file_open
             => ServicesVSResources.Collapse_metadata_on_file_open;
 
+        public static string Option_Collapse_empty_metadata_on_file_open
+            => ServicesVSResources.Collapse_empty_metadata_on_file_open;
```

Comment:
don't have a great alternative, but 'empty' isn't the greatest term.  maybe 'signature-only metadata file' or 'non-implementation metadata file'?

Reply:
oh.  i see, these are just the C# identifiers, not the contents :)